settings.gradle

```diff
@@ -1 +1 @@
-include ':parser'
+include ':parser', ':app'
```

Comment:
Did this so Android Studio recognizes the `app` module and the app can be run directly from AS.